src/test/java/org/swisspush/redisques/AbstractTestCase.java

```diff
@@ -69,6 +69,15 @@ protected void assertLockContent(TestContext context, String queuename, String e
         }
     }
 
+    protected boolean delay(long ms) {
+        try {
+            Thread.sleep(ms);
+        } catch (InterruptedException iex) {
+            return false;

```

Comment:
- https://www.yegor256.com/2015/10/20/interrupted-exception.html
- https://stackoverflow.com/questions/3976344/handling-interruptedexception-in-java
- https://softwareengineering.stackexchange.com/a/190535/silently-ignore-errors

Reply:
> it's just used in unit tests

In this case, my recommendation would be:
```java
catch( InterruptedException ex ){
    Thread.currentThread().interrupt();
    throw new RuntimeException(ex);
}
```